AsyncDisplayKit/Private/_ASPendingState.mm

```diff
@@ -207,6 +209,7 @@ - (instancetype)init
   contents = nil;
   isHidden = NO;
   needsDisplayOnBoundsChange = NO;
+  allowsGroupOpacity = YES;
```

Comment:
We have to read the default value from a `UIView` or from the `Info.plist`of the project. Addressed